cmd/operator-sdk/new/cmd.go

```diff
@@ -165,7 +161,32 @@ func newFunc(cmd *cobra.Command, args []string) error {
 			log.Fatal(err)
 		}
 	case projutil.OperatorTypeHelm:
-		if err := doHelmScaffold(); err != nil {
+		// create the project dir
+		err := os.MkdirAll(projectName, 0755)
+		if err != nil {
+			log.Fatal(err)
+		}
+		// go inside of the project dir
+		err = os.Chdir(filepath.Join(projutil.MustGetwd(), projectName))
+		if err != nil {
+			log.Fatal(err)
+		}
+
+		cfg := input.Config{
+			AbsProjectPath: filepath.Join(projutil.MustGetwd()),
+			ProjectName:    projectName,
+		}
+
+		createOpts := helm.CreateChartOptions{
```

Comment:
I realize that Helm project initialization is different from that for Go in that the SDK scaffolds an API before running `add api`, but does it now make sense to align how APIs are created with the way they are for Go scaffolds? The UX would benefit from differentiating project initialization from API creation.

Reply:
This would break UX for legacy projects so this actually isn't something that could have been done in this PR.

Reply:
I think that makes sense all commands follow the same standard.  
It is NOT a breaking change since nothing was changed in this PR.  We are just organizing the implementation as aligned before to make easier we do the reviews.

Reply:
I actually asked this question when I was originally adding the Helm operator implementation into the SDK.

The reason we allow Ansible and Helm operators to add APIs during project initialization is that we can basically import everything necessary (either a helm chart or ansible playbook/role) to have a fully working operator in one command.

To try and get folks onboarded, there was a desire to make it super easy to get up and running with ansible and helm.

We should chat with @fabianvf and @dmesser to see if it would make sense to keep this one-step UX or to align more to Go operators' two-step UX.

Reply:
Continued thread: https://github.com/operator-framework/operator-sdk/pull/3193#discussion_r438895772.